src/olympia/reviewers/templates/reviewers/includes/version.html

```diff
@@ -18,7 +18,7 @@
     </span>
     {% endif %}
 
-    {% if addon.block and addon.block.is_version_blocked(version.version) %}
+    {% if version.is_blocked %}
```

Comment:
Drive-by change after I found it in a search and noticed it wasn't using the shortcut function.